README.md

```diff
@@ -1,198 +1,163 @@
 
-client.js[0].toHTML();
-// <script type="module" src="http://<asset server>/finn/js/my-app/1.0.0/main/index.js"><script>
+In addition this module can also download the import maps defined in Eik config and provide these for inclusion in an application.
 
-client.js[1].toHTML();
-// <script src="http://<asset server>/finn/js/my-app/1.0.0/ie11/index.js"><script>
+The following will use the information in Eik config and provide an absolute URL to a file on an Eik server:
 
-client.css[0].toHTML();
-// <link rel="stylesheet" type="text/css" href="http://<asset server>/finn/js/my-app/1.0.0/main/index.css">
-```
+```js
+import EikNodeClient from '@eik/node-client';
 
-### in development mode the client will build an object pointing to development assets provided
+const client = new EikNodeClient({
+    development: false,
+    base: 'http://localhost:8080/public'        
+});
 
-```js
-const client = new Client({ js: '/assets/scripts.js', css: '/assets/styles.css' development: true });
+await client.load();
 
-// client.js will be an array of the form [{ type: 'module', value: '/assets/script.js' }]
-// client.css will be an array of the form [{ type: 'text/css', value: '/assets/styles.css' }]
+// Will, for example, output: 
+// {
+//   integrity: sha512-zHQjnDpMW7IKVyTpT9cOPT1+xhUSOcbgXj6qHCPSPu1CbQfgwDEsIniXU54zDIN71zqmxLSp3hfIljpt69ok0w==
+//   value: https://cdn.eik.dev/pkg/mymodue/2.4.1/path/script.js   
+// }
+client.file('/path/script.js')
 ```
 
-It's up to you to make sure that these assets are available to the app.
-In an express app you might use `express-static` to serve the assets with your app
-
-_Example: Express app using express-static_
+The following is the same as above but in development mode. The output will then be based on the vaule set for `base`:
 
 ```js
-app.use('/assets', express.static('assets'));
+import EikNodeClient from '@eik/node-client';
 
-const client = new Client({
-    js: '/assets/scripts.js',
-    css: '/assets/styles.css'
-    development: true
+const client = new EikNodeClient({
+    development: true,
+    base: 'http://localhost:8080/public'        
 });
-```
-
-Or you might use webpack or some other bundling system that can also serve the assets in development mode for you
-(remembering to set appropriate CORS headers)
 
-_Example: Setup when using Webpack dev server_
+await client.load();
 
-```js
-const client = new Client({
-    js: 'http://localhost:8080/scripts.bundle.js',
-    development: true,
-});
+// Will, for example, output: 
+// {
+//   integrity: null
+//   value: http://localhost:8080/public/path/script.js
+// }
+client.file('/path/script.js')
```

Comment:
`fileUrl`, `httpFileDescriptor`, `assetFor` might be better than just `file` as you are not getting a file back?